lib/Consumer.js

```diff
@@ -419,6 +433,76 @@ class Consumer extends EnhancedEventEmitter
 	}
 
 	/**
+	 * Enables periodic stats retrieval.
+	 *
+	 */
+	enableStats()
+	{
+		logger.debug('enableStats()');
+
+		if (this._closed)
+		{
+			logger.error('enableStats() | Consumer closed');
+
+			return;
+		}
+
+		if (this._statsEnabled)
+		{
+			return;
+		}
+
+		this._statsEnabled = true;
+
+		this._statsInterval = setInterval(() =>
+		{
+			if (this._paused || !this.enabled)
```

Comment:
This is wrong, there is no `this._paused` in Consumer. Use `this.paused` getter:

```js
	get paused()
	{
		return this._locallyPaused || this._remotelyPaused || this._sourcePaused;
	}
```